src/App.jsx

```diff
@@ -129,22 +53,20 @@ export default function App() {
 		setData([...dataClone]);
 	};
 
-	const handleStudentCheck = (studentId, sectionId, classId) => {
+	const handleStudentCheck = (studentIndex, sectionIndex, classIndex) => {
 		let dataClone = [...data];
-		let classIndex = dataClone.findIndex((cls) => cls.id === classId);
 		let sections = dataClone[classIndex].sections;
-		let sectionIndex = sections.findIndex((sec) => sec.id === sectionId);
-		let students = sections[sectionIndex].students.map((std) => {
-			if (std.id === studentId) {
+		let students = sections[sectionIndex].students.map((student, index) => {
```

Comment:
can we do this all in this single line . By that i mean u have student index and section index so that means u can directly access the student right? Is it possible to that?

Reply:
Yes, we can do that. I will check this line again. Thanks for mentioning.